array.c

```diff
@@ -7849,15 +7849,15 @@ rb_ary_deconstruct(VALUE ary)
  *
  *  You can convert certain objects to Arrays with:
  *
- *  - \Method #Array.
+ *  - \Method {Array}[Kernel.html#method-i-Array].
```

Comment:
What's the reason of this change? Even if changing references to explicit, we should use `rdoc-ref:` scheme for such cases (e.g. `[rdoc-ref:Kernel#Array]`), but I think `#Array` [works fine currently](https://docs.ruby-lang.org/en/3.1/Array.html#class-Array-label-Creating+Arrays)?..

Reply:
Reverted.